docker/confd/templates/config-single.xml

```diff
@@ -0,0 +1,74 @@
+<Configuration>
+    <Server
+        maxThreads="20"
+        port="1080"/>
+
+    <Pipeline
+        name="outbound"
+        root="/data/out">
+        <DicomImportService
+            class="org.rsna.ctp.stdstages.DicomImportService"
+            name="DicomImportService: outbound"
+            port="{{ or (getenv "CTP_OUTBOUND_PORT") "11111" }}"
+            calledAETTag="00097770"
+            callingAETTag="00097772"
+            connectionIPTag="00097774"
+            logConnections="yes"
+            timeTag="00097776"
+            quarantine="quarantines/DicomImportService"
+            root="roots/DicomImportService">
+            <accept callingAET="{{ getenv "CTP_LOCAL_AET" }}"/>
+        </DicomImportService>
+        <DicomFilter
+            name="DicomFilter"
+            class="org.rsna.ctp.stdstages.DicomFilter"
+            root="roots/DicomFilter"
+            script="scripts/dicom-filter.script"
+            quarantine="quarantines/DicomFilter" />
+        <DicomAnonymizer
+            name="DicomAnonymizer"
+            id="anonymization: outbound"
+            class="org.rsna.ctp.stdstages.DicomAnonymizer"
+            root="roots/DicomAnonymizer"
+            script="scripts/AnonymizeOutbound.xml"
```

Comment:
Good point, will add. Also, this should reference "AnonymizeOut.xml", and not "AnonymizeOutbound.xml".

Reply:
As discussed offline, we should think about an improved way of handling anonymization scripts (and configuration in general), since we now have to potentially change 3 different scripts at runtime, which complicates `run.sh`.

I would prefer it if we could merge all 3 configuration templates (single, proxy and cloud) into one, and also use a single anonymization script for all of them.

Anyway, I updated this `config-single.xml` template to reference `DicomServiceAnonymizer.script`, and make it consistent with the old `config.xml` we use for current premise deployments.